src/classes/User.as

```diff
@@ -229,6 +236,77 @@ package classes
             this.averageRank = (rankTotal / _gvars.TOTAL_PUBLIC_SONGS);
         }
 
+        ///- Determine what songs make up the user's Skill Rating
+        public function getUserSkillRatingData():void
+        {
+            for (var key:int in this.level_ranks)
+            {
+                var levelRank:Object = this.level_ranks[key];
+                
+                //Calculate the AAA Equiv for the scores on the song if greater than 0
+                if (levelRank.score != 0)
```

Comment:
I suppose this should actually be checking `> 0` instead of not 0 if the comment specifies above it.